src/cif.cc

```diff
@@ -282,6 +282,10 @@ freesasa_cif_structure_array(std::FILE *input,
 
             ss.reserve(n_new_chains);
             for (auto &chain_name : *chain_names) {
+                if (models[i].find_chain(chain_name)->get_polymer().size() == 0) {
```

Comment:
I think this filtering should be done on the `freesasa_structure`, it's the `options` and `classifier` that decide which atoms are kept. So something like

```c
freesasa_structure* structure = freesasa_structure_from_chain(/*...*/);
if (freesasa_structure_n(structure) === 0) {
  free(structure);
  continue;
}
ss.emplace_back(/*...*/);

Reply:
Yeah I agree with you. The freesasa values seem to match the PDB ones better for some reason. I just pushed.